build-system/replace-secrets.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2020 Google Inc.
```

Comment:
`Copyright 2020 The AMP HTML Authors. All Rights Reserved.`

Reply:
:eyes:

Reply:
Ooof. Copied this from another file, but idr which :thinking: Fixed.